frontend/frontend/index.html

```diff
@@ -0,0 +1 @@
+<!doctype html><html lang="en"><head><meta charset="UTF-8"><meta name="viewport" content="width=device-width,initial-scale=1"><meta http-equiv="X-UA-Compatible" content="ie=edge"><base href="/"/><title>Certificate Manager</title><link rel="icon" href="favicon.ico" type="image/x-icon"><meta http-equiv="Content-Security-Policy" content="default-src * self blob: data: gap:; style-src * self 'unsafe-inline' blob: data: gap:; script-src * 'self' 'unsafe-eval' 'unsafe-inline' blob: data: gap:; object-src * 'self' blob: data: gap:; img-src * self 'unsafe-inline' blob: data: gap:; connect-src self * 'unsafe-inline' blob: data: gap:; frame-src * self blob: data: gap:;"><script defer="defer" src="/frontend/main.js"></script><link href="/frontend/main.css" rel="stylesheet"></head><body><div id="root"></div></body></html>
```

Comment:
Hmm is this mistake? Shouldn't content of frontend directory be generated on build instead of commited to git?